lib/engines/content_object_store/features/object_store_feature_flag.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Create a content object when feature flag is disabled
+
+  Background:
```

Comment:
is this enough of a test for the feature flag working when disabled? Not sure how much coverage we need 🤔

Reply:
I think this is a reasonable 👍 

How might we increase this if we wanted to? We could write integration tests for each path, that might better catch any unexpected changes, eg `before_action :check_object_store_feature_flag, except: delete`.